frontend/src/views/Reporting/makeFullReport.ts

```diff
@@ -272,15 +274,42 @@ const generateManureAndCompostUse = (
       3: { cellWidth: pageWidth * 0.17 },
       4: { cellWidth: 'auto' },
     },
-    body: nmpFileYear.nutrientAnalyses.map((analysis) => [
-      analysis.manureName,
-      analysis.uniqueMaterialName,
-      `${printNum(analysis.annualAmount)} ${analysis.solidLiquid === 'Solid' ? 'tons' : 'US gallons'}`,
-      'TBC',
-      // TODO: Add logic for this footnote:
-      // "If the amount remaining is less than 10% of the annual amount, then the amount remaining is insignificant (i.e. within the margin of error of the calculations)"
-      'TBC',
-    ]),
+    body: nmpFileYear.nutrientAnalyses.map((analysis) => {
+      let landApplied = 'N/A';
+      let amountRemaining = 'N/A';
+
+      if (materialRemainingData) {
+        const allMaterials = [
+          ...materialRemainingData.appliedStoredManures,
+          ...materialRemainingData.appliedImportedManures,
+        ];
+        const matchingMaterial = allMaterials.find(
+          (material) => material.sourceUuid === analysis.sourceUuid,
+        );
+
+        if (matchingMaterial) {
+          const unit = analysis.solidLiquid === 'Solid' ? 'tons' : 'US gallons';
+
+          // Format land-applied with percentage
+          landApplied = `${printNum(matchingMaterial.totalApplied)} ${unit} (${matchingMaterial.wholePercentApplied}%)`;
+
+          // Format amount remaining - show "None" if less than 10%, otherwise show amount with percentage
+          if (matchingMaterial.wholePercentRemaining < 10) {
+            amountRemaining = 'None';
+          } else {
+            amountRemaining = `${printNum(matchingMaterial.totalAnnualManureRemainingToApply)} (${matchingMaterial.wholePercentRemaining}%)`;
```

Comment:
Was going to say that the unit is missing, but I see that's how it is in old NMP lol